terasoluna-gfw-common-libraries/terasoluna-gfw-web/src/main/java/org/terasoluna/gfw/web/exception/ExceptionLoggingFilter.java

```diff
@@ -159,7 +159,7 @@ protected void logServletException(ServletException ex,
      */
     protected void logRuntimeException(RuntimeException ex,
             ServletRequest request, ServletResponse response) {
-        exceptionLogger.log(ex);
+        exceptionLogger.error(ex);
     }

```

Comment:
- [ ] Is this necessary? https://github.com/terasolunaorg/terasoluna-gfw/blob/769907c6165b622e73a30692b3ddeb9d7e9fd1e9/terasoluna-gfw-common-libraries/terasoluna-gfw-common/src/main/java/org/terasoluna/gfw/common/exception/ExceptionLogger.java#L272-L282